runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -146,6 +158,32 @@ public void onVarbitChange(VarbitChanged event)
 			removeGameTimer(PRAYER_ENHANCE);
 			lastRaidVarb = raidVarb;
 		}
+
+		int inWilderness = client.getVar(Varbits.IN_WILDERNESS);
+		if (lastWildernessVarb != inWilderness
+			&& client.getGameState() == GameState.LOGGED_IN
+			&& !loggedInRace)
+		{
+			if (!WorldType.isPvpWorld(client.getWorldType())
+				&& inWilderness == 0)
+			{
+				log.debug("Left wilderness in non-PVP world, clearing Teleblock timer.");
+				removeTbTimers();
+			}
+			lastWildernessVarb = inWilderness;
+		}
+	}
+
+	@Subscribe
+	public void onWidgetHiddenChanged(WidgetHiddenChanged event)
+	{
+		Widget widget = event.getWidget();
+		if (WorldType.isPvpWorld(client.getWorldType())
+			&& WidgetInfo.TO_GROUP(widget.getId()) == WidgetInfo.PVP_CONTAINER.getGroupId()
+			&& WidgetInfo.TO_CHILD(widget.getId()) >= WidgetInfo.PVP_CONTAINER.getChildId())
```

Comment:
is this >= really necessary? seems wonky

Reply:
I guess not. Checking the group is probably sufficient.